echo/server/prod-worker.sh

```
@@ -1,2 +1,18 @@
+
+PROCESSES=${NETWORK_WORKER_PROCESSES:-2}
+THREADS=${NETWORK_WORKER_THREADS:-20}
+
+echo "Configuration:"
+echo "  Processes: $PROCESSES | Threads: $THREADS"
+echo "  Capacity per pod: $((PROCESSES * THREADS)) concurrent tasks"
+echo "📊 Scale with K8s replicas"
+
+exec dramatiq-gevent \
+  --queues network \
+  --processes $PROCESSES \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)